packages/server/scripts/test.sh

```diff
@@ -3,12 +3,12 @@ set -e
 
 if [[ -n $CI ]]
 then
-  export NODE_OPTIONS="--max-old-space-size=4096 --no-node-snapshot"
+  export NODE_OPTIONS="--max-old-space-size=4096 --no-node-snapshot $NODE_OPTIONS"
```

Comment:
This allows VSCode's "auto attach" feature to work. It passes values into `NODE_OPTIONS` that allows it to attach a debugger to any node process started from a VSCode terminal.